l2tdevtools/build_helpers/wheel.py

```diff
@@ -0,0 +1,236 @@
+          projects, which is used to determine the properties of dependencies.
+    """
+    super(WheelBuildHelper, self).__init__(
+        project_definition, l2tdevtools_path, dependency_definitions)
+    self.architecture = None
+
+    # Note that platform.machine() does not indicate if a 32-bit version of
+    # Python is running on a 64-bit machine, hence we need to use
+    # platform.architecture() instead.
+    architecture = platform.architecture()[0]
+    if architecture == '32bit':
+      self.architecture = 'win32'
+    elif architecture == '64bit':
+      self.architecture = 'win_amd64'
+
+  def _GetWheelFilenameProjectInformation(self, source_helper_object):
+    """Determines the project name and version used by the wheel filename.
+
+    Args:
+      source_helper_object (SourceHelper): source helper.
+
+    Returns:
+      tuple: contains:
+
+        * str: project name used by the wheel filename.
+        * str: project version used by the wheel filename.
+    """
+    if self._project_definition.wheel_name:
+      project_name = self._project_definition.wheel_name
+    elif self._project_definition.setup_name:
+      project_name = self._project_definition.setup_name
+    else:
+      project_name = source_helper_object.project_name
+
+    project_version = source_helper_object.GetProjectVersion()
+
+    return project_name, project_version
+
+  def _MoveWheel(self, source_helper_object):
+    """Moves the wheel from the dist sub directory into the build directory.
+
+    Args:
+      source_helper_object (SourceHelper): source helper.
+
+    Returns:
+      bool: True if the move was successful, False otherwise.
+    """
+    project_name, _ = self._GetWheelFilenameProjectInformation(
+        source_helper_object)
+
+    source_directory = source_helper_object.GetSourceDirectoryPath()
+    if not source_directory:
+      logging.info('Missing source directory of: {0:s}'.format(
+          source_helper_object.project_name))
+      return False
+
+    filenames_glob = os.path.join(
+        source_directory, 'dist', '{0:s}-*-*-*-*.whl'.format(project_name))
+    filenames = glob.glob(filenames_glob)
+
+    if len(filenames) != 1:
+      logging.error('Unable to find wheel file: {0:s}.'.format(filenames_glob))
+      return False
+
+    _, _, wheel_filename = filenames[0].rpartition(os.path.sep)
+    if os.path.exists(wheel_filename):
+      logging.warning('Wheel file already exists.')
+    else:
+      logging.info('Moving: {0:s}'.format(filenames[0]))
+      shutil.move(filenames[0], '.')
+
+    return True
+
+  def CheckBuildDependencies(self):
+    """Checks if the build dependencies are met.
+
+    Returns:
+      list[str]: build dependency names that are not met or an empty list.
+    """
+    missing_packages = []
+    for package_name in self._project_definition.build_dependencies:
+      if package_name not in ('fuse', 'libcrypto', 'zlib'):
```

Comment:
Please extract these to a class constant, and add a comment as to why they're not considered as missing deps.

Reply:
done